ui/src/app/certificate_requests/row.tsx

```diff
@@ -100,9 +106,20 @@ export default function Row({ id, csr, certificate, ActionMenuExpanded, setActio
                     </span>
                 </td>
                 <td id="expanded-row" className="p-table__expanding-panel" aria-hidden={detailsMenuOpen ? "false" : "true"}>
-                    <div className="row">
-                        <div className="col-8">
-                            <p><b>Common Name</b>: {csrObj.subjects.find((e) => e.type == "Common Name")?.value}</p>
+                    <div className="col-8">
+                        <div className="certificate-info">
+                            <h4>Basic Information</h4>
+                            {getFieldDisplay("Common Name", csrObj.commonName, "N/A")}
+                            {getFieldDisplay("Organization", csrObj.organization, "N/A")}
+                            {getFieldDisplay("Country", csrObj.country, "N/A")}
+                            {getFieldDisplay("Locality", csrObj.locality, "N/A")}
+                            {getFieldDisplay("Email Address", csrObj.emailAddress, "N/A")}
+                            <p><b>Certificate request for a certificate authority</b>: {csrObj.is_ca ? "Yes" : "No"}</p>
+                        </div>
+                        <div className="certificate-info">
+                            <h4>Subject Alternative Names</h4>
```

Comment:
I don't know about separating the fields into those arbitrary categories (Basic Information and Subject Alternative Names). For now I would recommend going with showing the fields in an order that makes sense (ex. having subject alternative names should likely be quite high, right after common name). 

In the future we can change this based on user feedback if necessary

Reply:
Removed and re-ordered.